doc/system-upgrade.rst

```diff
@@ -0,0 +1,207 @@
+
+``clean``
+    Remove previously-downloaded data. This happens automatically at the end of
+    a successful upgrade.
+
+``log``
+    Used to see a list of boots during which an upgrade was attempted, or show
+    the logs from an upgrade attempt. The logs for one of the boots can be shown
+    by specifying one of the numbers in the first column. Negative numbers can
+    be used to number the boots from last to first. For example, ``log --number=-1`` can
+    be used to see the logs for the last upgrade attempt.
+
+-------
+Options
+-------
+
+``--releasever=VERSION``
+    REQUIRED. The version to upgrade to. Sets ``$releasever`` in all enabled
+    repos. Usually a number, or ``rawhide``.
+
+``--downloaddir=<path>``
+    Redirect download of packages to provided ``<path>``. By default, packages
+    are downloaded into (per repository created) subdirectories of
+    /var/lib/dnf/system-upgrade.
+
+``--distro-sync``
+    Behave like ``dnf distro-sync``: always install packages from the new
+    release, even if they are older than the currently-installed version. This
+    is the default behavior.
+
+``--no-downgrade``
+    Behave like ``dnf update``: do not install packages from the new release
+    if they are older than what is currently installed. This is the opposite of
+    ``--distro-sync``. If both are specified, the last option will be used. The option cannot be
+    used with the ``offline-distrosync`` command.
+
+``--number``
+    Applied with ``log`` subcommand will show the log specified by the number.
+
+-----
+Notes
+-----
+
+``dnf system-upgrade reboot`` does not create a "System Upgrade" boot item. The
+upgrade will start regardless of which boot item is chosen.
+
+The ``DNF_SYSTEM_UPGRADE_NO_REBOOT`` environment variable can be set to a
+non-empty value to disable the actual reboot performed by ``system-upgrade``
+(e.g. for testing purposes).
+
+Since this is a DNF plugin, options accepted by ``dnf`` are also valid here,
+such as ``--allowerasing``.
+See :manpage:`dnf(8)` for more information.
+
+The ``fedup`` command is not provided, not even as an alias for
+``dnf system-upgrade``.
+
+----
+Bugs
+----
+
+Upgrading from install media (e.g. a DVD or .iso file) currently requires the
+user to manually set up a DNF repo and fstab entry for the media.
+
+--------
+Examples
+--------
+
+Typical upgrade usage
+---------------------
+
+``dnf --refresh upgrade``
+
+``dnf system-upgrade download --releasever 26``
+
+``dnf system-upgrade reboot``
+
+Show logs from last upgrade attempt
+-----------------------------------
+
+``dnf system-upgrade log --number=-1``
+
+--------------
+Reporting Bugs
+--------------
+
+Bugs should be filed here:
+
+  https://bugzilla.redhat.com/
+
+For more info on filing bugs, see the Fedora Project wiki:
+
+  https://fedoraproject.org/wiki/How_to_file_a_bug_report
+
+  https://fedoraproject.org/wiki/Bugs_and_feature_requests
```

Comment:
Don't we have DNF specific documentation to reference here instead?

Reply:
The DNF wiki was dropped and it's not going to be redeployed as our goal is to move to DNF5. Currently there is no living DNF-specific link related to bugs filing as I know, so I'll keep the existing doc state copied from `dnf-plugins-extras`.